lib/front-end-connection.js

```diff
@@ -1,131 +1,46 @@
 'use strict';
```

Comment:
This file got much simpler. We don't have to create a server, handle websocket errors, JSON.parse etc.